avm/res/document-db/database-account/tests/e2e/role/main.test.bicep

```diff
@@ -0,0 +1,60 @@
+targetScope = 'subscription'
+
+metadata name = 'Deploying with a key vault reference to save secrets'
```

Comment:
This metadata value is a copy paste of another test. Can we write a meaningful one?